src/build/__tests__/build-test.js

```diff
@@ -9,15 +9,30 @@ import {
   GraphQLBoolean,
   GraphQLID,
 } from 'graphql/type';
+import { DirectiveLocation, GraphQLDirective } from 'graphql/type/directives';
```

Comment:
nit: These 2 imports are available from `graphql/type` so you could just add them at the end of the import statement above.